examples/main_test.go

```diff
@@ -6,7 +6,10 @@ import (
 )
 
 func TestMain(m *testing.M) {
-	os.Setenv("PULUMI_TEST_OWNER", "service-provider-test-org")
-	os.Setenv("PULUMI_TEST_USE_SERVICE", "true")
+	// Only set PULUMI_TEST_OWNER if not already set by environment
+	if os.Getenv("PULUMI_TEST_OWNER") == "" {
+		_ = os.Setenv("PULUMI_TEST_OWNER", "service-provider-test-org")
+	}
+	_ = os.Setenv("PULUMI_TEST_USE_SERVICE", "true")
 	m.Run()
```

Comment:
Nit: assign the result of `os.Getenv("...")` to variables and pass them down to `m.Run()`? The comment would then not be necessary.